x-pack/plugins/task_manager/server/queries/task_claiming.test.ts

```diff
@@ -909,7 +909,7 @@ if (doc['task.runAt'].size()!=0) {
         mockInstance({
           id: 'aaa',
           runAt,
-          taskType: 'foo',
+          taskType: 'yawn',
```

Comment:
just curious if these changes were for a specific reason :)

Reply:
:)
Initially i filtered the invalid tasks in task_claiming and these tests failed because the only registered tasks in this test file are: `report`, `dernstraight` and `yawn` (on the line 66 in this file), so i replaced the taskNames with one of the registereds.
I reverted that code but using a registered taskName here still less confusing.